### ISSUE TITLE
AutoScroll toggle loses focus

### ISSUE THREAD

**Issue by nirmalsahoo** [microsoft/vscode#97128]
**Environment Details:**
VSCode Version : 1.44.2
OS Version : Win10
**Additional Details:**
GitHubTags: #A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCode; #MAS2.4.3; #MAS2.4.7; #A11ySev2;#CAITestPass_May2020;#Benchmark;#Visual Studio Code;
**Repro Steps:**
1)Open Visual studio code editor
2)Press ctrl+shift+M.
3)Validate it's possible to navigate between items in the problems view
4)Navigate to the output tab and then navigate to "Turn Auto Scrolling On/Off".
5)Check the keyboard focus is logical after click on "Turn Auto Scrolling On/Off".
**Actual:**
While navigating through keyboard, Keyboard focus is invisible and not remain on the button after click on "Turn Auto Scrolling On/Off".

**Expected:**
Keyboard focus should remain on the button after click enter on the "Turn Auto Scrolling On/Off" button and should be visible.

**User Impact:**
Keyboard and low vision user unable to get the focus in logical way.
**Attachment For Reference:**

[97128_A11y_VSCode_Output Panel_Keyboard.zip](https://github.com/microsoft/vscode/files/4592115/97128_A11y_VSCode_Output.Panel_Keyboard.zip)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
I can repro this issue. The problem is that the Output lock action once triggered loses focus - probably due to the toggling.
@sandy081 is there some way that we preserve focus on the output lock action once it is toggled?

**Comment by sandy081** [comment]
@isidorn I think I am not changing the focus after clicking on this action. Could be the action item is not retaining the focus. BTW this is a toggle action registered using menu actions, that could be the reason?

**Comment by isidorn** [comment]
It is not because ot the action item I think, but because the whole icon / action get swapped based on state.

**Comment by sandy081** [comment]
Yes that's because I am using toggle-able action

https://github.com/microsoft/vscode/blob/ff118340b561fea6184172ffaf3bbf859b21055b/src/vs/workbench/contrib/output/browser/output.contribution.ts#L150-L174

Shall we have to fix at action layer or locally?

**Comment by isidorn** [comment]
@sandy081 ideally we would fix at the action layer, if the action had focused and is toggled -> the new action should have focus as well.

**Comment by sandy081** [comment]
let me know if there is something I have to do otherwise I can unassign myself?

**Comment by isidorn** [comment]
Let me unassign you. Thanks!

**Comment by isidorn** [comment]
I have changed the actionBar such that the focus should now be preserved when actions get toggled.

To verify:
1. Focus on the output view title actions and trigger the Scroll Lock action via space or enter
2. After the action has toggled focus should stay on it
